### ISSUE TITLE
Cannot import csv

### ISSUE THREAD

**Issue by markusd112** [RubenGees/Vocabletrainer#3]
I have filled an Excel file with my existing words (Column 1: italian word, 2: german translation, 3: lection) as described in the help.

But always an error occurs: "Import fehlgeschlagen, Fehler in Zeile: 0"

Is there any additional information or example available, how to import vocabularies?

**Comment by markusd112** [comment]
The problem was that excel uses an ";" as separation character. I haven't found any way to change this in excel or windows. So I have get the import worked, but now I have the next problem: all "Umlaute" aren't imported correctly. What is the excat way to import the files?

**Comment by rubengees** [comment]
Yeah I know about that problem. Many people try to import from excel, but I never intended them to do that. My thought was, that you write your vocables into a plane txt file and save it as csv.
About the "Umlaute" (I'm actually german, too :) ): You have to save the file with the encoding "UTF-8". On Windows you can find that at the save options of the default txt editor.

**Comment by markusd112** [comment]
For me it was nearby to use a spreadsheet program to bring it into the right form to import in "Vocabletrainer". It was an already existing vocabulary list: it has been exported from phase 6 software.
I think a helpful way would be to simply add some more information how to import the list, including which charset to be use.
Another suggestion would be to support xls files to import. Another vocable trainer I have tried does it this way: you can save an xls template out of the app, fill it with your vocables and then import it into the app. That worked very easy and was very intuitiv. Additionally in the excel template there were some additional hints, in which columns to put the german words and the translations and the unit names.
Please remember: I don't want to beef (your "...I never intended them to do that"). I try to help you to improve vocabletrainer by giving some feedback about my problems as newbie ;-).

**Comment by rubengees** [comment]
Yes, I didn't intend to be rude. The import from xls files would be a good thing of course, but it is somehow difficult, because of all the different formats from different excel versions. You are right that the help for the import is not very good, I really need to improve that...
